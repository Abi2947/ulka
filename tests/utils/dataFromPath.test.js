const dataFromPath = require('../../src/utils/dataFromPath')
const absolutePath = require('../../src/utils/absolutePath')

describe('Datafrompath function', () => {
  test('should get data from path', () => {
    const data = dataFromPath(
      absolutePath('/tests/resources/ulka_test/ulka-config.js')
    )
    delete data.path
    expect(data).toMatchSnapshot()
  })
})
