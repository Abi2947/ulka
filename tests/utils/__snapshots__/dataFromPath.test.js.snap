// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Datafrompath function should get data from path 1`] = `
Object {
  "data": "module.exports = {
  siteMetaData: {
    title: 'UlkaJs',
    description: 'This is a default starter for ulka js'
  },
  buildPath: 'build',
  pagesPath: 'pages',
  templatesPath: 'templates',

  contents: {
    path: 'contents',
    generatePath: 'blog',
    template: 'blog.ulka',
    preParse: [],
    postParse: []
  }
}
",
}
`;
